go/constants.go

```diff
@@ -5,3 +5,4 @@ const CONTENT_TYPE = "application/json"
 
 const DEFAULT_EXPIRY_IN_SECONDS = int64(30)
 const MAINNET_CHAIN_ID = 1
+const TESTNET_CHAIN_ID = 11155111
```

Comment:
We can remove these two lines if we get L1 chain id from `GET /system/config`

Reply:
Ah yeah that's a good point!
Gonna go ahead and do this ;)

Reply:
Just pushed up and tested ;)